front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/index.ts

```diff
@@ -273,23 +273,16 @@ async function handler(
         keyAuth.getNonNullableWorkspace()
```

Comment:
Use [this link](https://gist.github.com/davidebbo/abf6085907d65518150adbc9362fe222/revisions?diff=unified&w) to compare from before my original changes.